src/store/SwapMeta.h

```diff
@@ -169,7 +168,6 @@ DeprecatedSwapMetaType(const RawSwapMetaType type)
         STORE_META_VALID = 7
     };
     return
-        // TODO: simplify with std::underlying_type_t when switching to C++14
```

Comment:
I removed this TODO after failing to simplify this code using std::underlying_type_t. This controversial code can be simplified, but that simplification is not related to C++14 AFAICT, so it is best to just drop that specific TODO.